tests/BuildSystem/Build/changed-commands.llbuild

```diff
@@ -18,8 +18,8 @@
 
 ##### VERSION-BEGIN-1 #####
 
-# CHECK-VERSION-1: echo output-1 > output-1
-# CHECK-VERSION-1: cat output-1 > output
+# CHECK-VERSION-1: echo\ output-1\ \>\ output-1
+# CHECK-VERSION-1: cat\ output-1\ \>\ output
```

Comment:
Why would these be escaped? That seems wrong the user should be able to copy paste them into terminal.